roles/horizon/tasks/main.yml

```diff
@@ -116,3 +116,9 @@
 
 - include: monitoring.yml tags=monitoring,common
   when: monitoring.enabled|default('True')|bool
+
+- name: add horizon customization modules
+  pip: name="git+https://github.com/davidcusatis/bluebox-horizon-customization#egg=bluebox-horizon-customization"
+       virtualenv="{{ openstack_source.virtualenv_base }}/horizon/"
+  notify:
+    - restart apache
```

Comment:
@j2sol @craigtracey sure, this seems like the best approach. but i want to double check because in order to add it to the python_dependencies, it seems i'd have to upload this to PyPI so pip can just pull it down right? i had the understanding Craig did not want me to put it on PyPI. unless i am misunderstanding how these python_dependencies are retrieved/installed.